docs/pages/using_dmx_merger.md

```diff
@@ -0,0 +1,328 @@
+merger_config.slots = slots;
+merger_config.slot_owners = slot_owners;
+
+// Initialize a merger and obtain its handle.
+sacn_dmx_merger_t merger_handle;
+etcpal_error_t result = sacn_dmx_merger_create(&merger_config, &merger_handle);
+
+// Mergers can later be destroyed individually.
+// Keep in mind that sacn_deinit() will destroy all mergers automatically.
+etcpal_error_t result = sacn_dmx_merger_destroy(merger_handle);
+```
+<!-- CODE_BLOCK_MID -->
+```cpp
+// These buffers are updated on each merger call with the merge results.
+// They must be valid as long as the merger is using them.
+uint8_t slots[DMX_ADDRESS_COUNT];
+sacn_source_id_t slot_owners[DMX_ADDRESS_COUNT];
+
+// Merger configuration used for the initialization of each merger:
+sacn::DmxMerger::Settings settings(slots, slot_owners);
+
+// Initialize a merger.
+sacn::DmxMerger merger;
+etcpal::Error result = merger.Startup(settings);
+
+// Mergers can later be destroyed individually.
+// Keep in mind that sacn_deinit() will destroy all mergers automatically.
+merger.Shutdown();
+```
+<!-- CODE_BLOCK_END -->
+
+## Adding and Removing Sources
+
+In order to input data into a merger, each source being merged must first be added to the merger.
+To do this, use the add source function, which takes the source's CID as input and provides a
+source handle on success, or an error code on failure. A source can later be removed individually,
+which also removes it from the merge output.
+
+<!-- CODE_BLOCK_START -->
+```c
+// Add a couple of sources, which are tracked with handles and CIDs.
+sacn_source_id_t source_1_handle, source_2_handle;
+EtcPalUuid source_1_cid, source_2_cid;
+// Initialize CIDs here...
+
+etcpal_error_t result;
+result = sacn_dmx_merger_add_source(merger_handle, &source_1_cid, &source_1_handle);
+result = sacn_dmx_merger_add_source(merger_handle, &source_2_cid, &source_2_handle);
+
+// Sources can later be removed individually, which updates the merger's output.
+result = sacn_dmx_merger_remove_source(merger_handle, source_1_handle);
+result = sacn_dmx_merger_remove_source(merger_handle, source_2_handle);
+```
+<!-- CODE_BLOCK_MID -->
+```cpp
+// Add a couple of sources, which are tracked with handles and CIDs.
+sacn_source_id_t source_1_handle, source_2_handle;
+etcpal::Uuid source_1_cid, source_2_cid;
+// Initialize CIDs here...
+
+// AddSource may also return an error, which is not checked for in this example.
+source_1_handle = merger.AddSource(source_1_cid).value();
+source_2_handle = merger.AddSource(source_2_cid).value();
+
+// Sources can later be removed individually, which updates the merger's output.
+merger.RemoveSource(source_1_handle);
+merger.RemoveSource(source_2_handle);
+```
+<!-- CODE_BLOCK_END -->
+
+## Inputting Levels and Priorities
+
+Different types of data can be fed into a merger: the NULL start code levels, per-address
+priorities, and universe priority. Inputting this data into a merger causes its merge output to
```

Comment:
There should be notes throughout this documentation that per-address priority is an ETC-specific sACN extension, and is disabled if the library is compiled with SACN_ETC_PRIORITY_EXTENSION=0.